index.bs

```diff
@@ -523,9 +526,14 @@ When this method is invoked, the user agent MUST execute the following algorithm
     any), to create a {{ClientData}} structure representing this request. Choose a hash algorithm for {{ClientData/hashAlg}} and
     compute the {{AuthenticationAssertion/clientDataJSON}} and <a>clientDataHash</a>.
 
-6. Initialize |issuedRequests| to an empty list.
+6. Initialize |issuedRequests| to an empty list and |currentlyAvailableAuthenticators| to empty lists.
```

Comment:
Remove the first "to an empty list"?

#347 expects that these are both [sets](https://infra.spec.whatwg.org/#sets) rather than lists.